recipes/xz/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "5.2.2" %}
+
+package:
+    name: xz
+    version: {{ version }}
+
+source:
+    fn:  xz-{{ version }}.tar.bz2
+    url: http://tukaani.org/xz/xz-{{ version }}.tar.gz
+    sha256: 73df4d5d34f0468bd57d09f2d8af363e95ed6cc3a4a86129d2f2c366259902a2
+
+build:
+    number: 0
+    skip: True  # [win]
+
+test:
+    commands:
+        - xz --help
+        - unxz --help
+        - lzma --help
+        - otool -L $PREFIX/lib/liblzma.5.dylib   # [osx]
+
+about:
+    home: http://tukaani.org/xz/
+    summary: The xz command is a very powerful program for compressing files
+    license: LGPL-2.1 and GPL-2.0
+
+extra:
+    recipe-maintainers:
+        - ccordoba12
```

Comment:
@ccordoba12 I stole this recipe from anaconda-recipes and it seems that you had done some work on it. Would you be OK to be a maintainer here?